docs/data/sdks/typescript-react-native/index.md

```diff
@@ -654,6 +655,16 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 });
 ```
 
+### Location
+
+The SDK will set the `country` based on network information. However, to remove permissions from the SDK that may not be required for all customers, we extracted precise location support from the core SDK. To use location you will now need to create an enrichment Plugin to set the location yourself. 
```

Comment:
Feel like it's a good place to explain how SDK and backend resolve location together. 
- Amplitude backends resolve location by the following order user-provided > ip resolved > lat/lng resolved
- For TS RN by default, the event payload will not have `country` field. However, it's filled up on backend based on IP address

Not sure if we should mention that 
- If COPPA is enabled which disables geo info tracking, you will not see `country` on "User & Look Up". 
- currently, this is no way to enable COPPA while have `country` tracked 
- the `getLocation` plugin tracks longitude and latitude which doesn't conform COPPA